### ISSUE TITLE
RFE: per-node pod symlinks

### ISSUE THREAD

**Issue by squeed** [openshift/must-gather#212]
TL;DR: create a folder for each node, symlink in pods running on that node.

Background: When debugging networking issues, we need to look at the logs for the network daemon running on a specific node. Of course, we can extract this from the pod's yaml, but this is somewhat tedious and potentially error-prone.

It would be useful if there was a way for the must-gather to make navigating on a per-node basis easier. It might look something like

`/per_node/NODENAME/pod/myns_podname.yaml -> /namespaces/myns/pods/podname/podname.yaml`
as well as
`/namespaces/myns/pods/per_node/NODENAME/podname.yaml -> /namespaces/myns/pods/podname/podname.yaml`

**Comment by oarribas** [comment]
You can use "o-must-gather" (https://github.com/kxr/o-must-gather/) with a must-gather, and you will be able to run some commands against the must-gather in the same way you do with "oc".

For the above case, you can execute:
~~~
omg use </path/to/must-gather/>
omg get pods -n <namespace> -o wide
omg get pod <pod_name> -n <namespace> -o yaml
omg logs -n <namespace> [-c <container_name>] <pod_name>
~~~

**Comment by squeed** [comment]
@oarribas interesting, that's clever. Can you get pods by fieldselector?

**Comment by oarribas** [comment]
> @oarribas interesting, that's clever. Can you get pods by fieldselector?

@squeed , I don't think so, not all the options in `oc` are implemented.

You can try to open a RFE for that in the "o-must-gather" proyect.
